integrationtests/src/test/java/io/kroxylicious/proxy/multitenant/MultiTenantIT.java

```diff
@@ -265,6 +266,12 @@ private void verifyConsumerGroups(String proxyAddress, String... expectedGroupId
         try (var admin = Admin.create(commonConfig(proxyAddress, Map.of()))) {
             var groups = admin.listConsumerGroups().all().get().stream().map(ConsumerGroupListing::groupId).toList();
             assertThat(groups).containsExactlyInAnyOrderElementsOf(Set.of(expectedGroupIds));
+            var describedGroups = admin.describeConsumerGroups(groups).all().get();
+            assertThat(describedGroups.size()).isEqualTo(expectedGroupIds.length);
+            // The consumer group comes back as "dead" when it doesn't exist, so we have to check that it's not dead
+            describedGroups.forEach((k, v) -> {
+                assertThat(v.state()).isNotEqualTo(ConsumerGroupState.DEAD);
```

Comment:
@showuon we found it really surprising that the describegroups uses the state ConsumerGroupState.DEAD to indicate a consumergroup that doesn't exist.  We were expecting something akin to `UnknownTopicOrPartitionException` that you get from describeTopics.

Reply:
Indeed! Checked the [KIP-40](https://cwiki.apache.org/confluence/display/KAFKA/KIP-40%3A+ListGroups+and+DescribeGroup#KIP40:ListGroupsandDescribeGroup-DescribeGroup), this is the expected behavior:

```
Group States: Below we list the possible group states and how it affects the associated metadata.

    Dead: There are no active members in the group. All other group metadata fields will be set to empty.
    Initializing: The group is loading metadata from its storage. All other group metadata fields will be set to empty and no member metadata will be returned.
    Rebalancing: The group is undergoing a rebalance. The generation, protocolType, and protocol will be set according to the prior generation. No member metadata will be returned.
    Stable: The group has a valid generation. Group and member metadata will be set based on the active generation.
```

I guess it's because this is a pretty early design (in 2016), so that's why.